packages/validate/lib/index.js

```diff
@@ -29,7 +29,7 @@ const versions = new Map([
 
 const keys = Array.from(versions.keys());
 
-const makeDesignator = JSON.stringify;
+const digest = JSON.stringify;
```

Comment:
This is nit-picky, but why not indicate that this is actually `JSON.stringify` and not some other implementation? Anybody will be familiar with `JSON.stringify` and not `digest`. The alias makes this a little harder to read.

Reply:
1bf8410d38e289fd8de9d043aa10622f7fa847c6